### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by derekeder** [datamade/usaddress-api#3]
related to #2 

Here's a fun address that's throwing things off:

1800 West¬¨‚Ä†1st Street
¬¨‚Ä†P.O. Box 410
Mlian

returns:
```python
UnicodeEncodeError: 'ascii' codec can't encode characters in position 2123-2127: ordinal not in range(128)
```

**Comment by cathydeng** [comment]
fixed on usaddress 0.3.4, tho it'll still raise a warning

**Comment by derekeder** [comment]
Oops, the issue is coming from `StringIO` not `usaddress`!
